pyttb/sptensor.py

```diff
@@ -671,12 +883,36 @@ def innerprod(
 
     def isequal(self, other: Union[sptensor, ttb.tensor]) -> bool:
         """
-        Exact equality for sptensors
+        Exact equality for sparse tensors, where all elements are exactly the
+        same in both tensors. Can be compared to another sparse tensor or
+        dense tensor.
```

Comment:
Same note on redundant types.
```suggestion
        same in both tensors.
```